lib/install/web.rb

```diff
@@ -0,0 +1,32 @@
+def apply_template!
+  if options[:database] == "postgresql" && options[:skip_test]
+    after_bundle do
+      gem_group :development, :test do
+        # TODO: Update once in `main`
+        gem "suspenders", github: "thoughtbot/suspenders", branch: "suspenders-3-0-0"
```

Comment:
Need to update this when we're ready to merge.
```suggestion
        gem "suspenders", github: "thoughtbot/suspenders", branch: "main"
```